sky/clouds/kubernetes.py

```diff
@@ -45,6 +46,8 @@ class Kubernetes(clouds.Cloud):
     # TODO(romilb): Make the timeout configurable.
     TIMEOUT = 10
 
+    _SUPPORTS_SERVICE_ACCOUNT_ON_REMOTE = True
```

Comment:
Is this needed?

Reply:
Yes, it's used by `cloud.supports_service_account_on_remote()` to tell whether to upload local cloud credential files